tests/pytests/test.py

```diff
@@ -1132,7 +1132,9 @@ def testExact(env):
     env.cmd('HSET', '{doc}:2', 'title', 'hello another world',
             'body', 'lorem ist ipsum lorem lorem')
 
-    for dialect in range(1,6):
+    MAX_DIALECT = set_max_dialect(env)
```

Comment:
Isn't this assignment redundant? (actually it is `MAX_DIALECT = MAX_DIALECT`)

Reply:
I tried to fix it, but it is needed because we are already using `from common import *`  which import `MAX_DIALECT` as a constant.
To import it as a property of common, I could add `import common`:
```sh
from common import *
import common
def test_000(env):
    common.set_max_dialect(env)
```
WDYT? 
Thanks.

Reply:
OK, so this is an assigment to a local copy of the variable. We can keep it and not use a fully qualified name `common.MAX_DIALECT`.